### DISCUSSION TITLE
请问「以词定字」功能如何单独使用？

### DISCUSSION THREAD

**Discussion by HUHARED** [LufsX/rime#31] [Q&A] [answered]
感谢LusfX。

**Comment by LufsX** [comment]
参见项目 [BlindingDark/rime-lua-select-character](https://github.com/BlindingDark/rime-lua-select-character)

**Reply by HUHARED** [reply]
抱歉， 我说的不清楚，上面这个是用的东风破，我参考它的内容不清楚如何在Windows平台使用。
之前尝试将rime.lua放到用户文件夹，然后新建一个lua文件夹并放置select_character.lua在里面，没有生效。

**Reply by LufsX** [reply, marked as answer]
把 `lua_processor@select_character_processor` 加到对应配置的 engine/processors/@before 0 里

以本配置的小鹤双拼为例既 https://github.com/LufsX/rime/blob/3a4840936c10837221f70f6deb01dd641bd95eff/lufs_flypy.schema.yaml#L35

同时 rime.lua 中要添加引用 https://github.com/LufsX/rime/blob/3a4840936c10837221f70f6deb01dd641bd95eff/rime.lua#L1-L3

**Reply by HUHARED** [reply]
可以了，非常感谢！
